src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -90,7 +90,7 @@ const Body = BaseFormWithPolling.extend(
           url: getAuthenticatorUrl('challenge'),
           method: 'POST',
           data: JSON.stringify({ challengeRequest }),
-          timeout: 3000 // authenticator should respond within 3000ms for challenge request
+          timeout: 300000 // authenticator should respond within 5 min (300000ms) for challenge request
```

Comment:
create a constant and put it in `Constants.js`.

Reply:
see if you need to change any `testcafe` tests..

Reply:
Can you please update the comment to give more context?